rn/Teacher/ios/Teacher/SpeedGrader/SubmissionGrader.swift

```diff
@@ -23,12 +23,100 @@ struct SubmissionGrader: View {
     let assignment: Assignment
     let submission: Submission
 
+    @Environment(\.appEnvironment) var env
+    @Environment(\.horizontalSizeClass) var horizontalSizeClass
+
+    @State var selectedDrawerTab: DrawerTab = .grades
+    @State var drawerState: DrawerState = .min
+
+    var bottomInset: CGFloat { env.window?.safeAreaInsets.bottom ?? 0 }
+
     var body: some View {
+        if horizontalSizeClass == .compact {
```

Comment:
This isn't exactly what we want because we want the `graderWithDrawer` on iPad in portrait (which is .regular). We may want to stick with the width check that we use in RN?

Do you want to fix this now or in a later PR? With it in development I don't care either way.

Reply:
I can fix it in a subsequent PR. That screen is far from done.